synapse/rest/client/v2_alpha/auth.py

```diff
@@ -177,7 +177,10 @@ async def on_GET(self, request, stagetype):
                 )
 
             elif self._saml_enabled:
-                client_redirect_url = b""
+                # Some SAML identity providers (e.g. Google) require a
+                # RelayState parameter on requests. It is not necessary here, so
+                # pass in a dummy redirect URL (which will never get used).
+                client_redirect_url = b"unused"
```

Comment:
Does google do anything with this? Or is it just echoed back?

Reply:
I believe it is just echoed back, a common use seems to be for the "url you want to end up at after login", many examples (e.g. Auth0, Okta docs) use both absolute and relative URLs (things like https://my-site.com/the/resource/you/want as well as just /the/resource/you/want). I do not believe this is usually shown to the user during login in anyway.